cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -357,18 +370,28 @@ def get_active_ceph_mgr(self):
             monitoring_endpoint_port = self.DEFAULT_MONITORING_ENDPOINT_PORT
 
         try:
+            failed_ip = monitoring_endpoint_ip
             monitoring_endpoint_ip = self._convert_hostname_to_ip(
                 monitoring_endpoint_ip)
+            # collect all the 'stand-by' mgr ips
+            mgr_ips = []
+            for each_standby_mgr in standby_mgrs:
+                failed_ip = each_standby_mgr
+                mgr_ips.append(
+                        self._convert_hostname_to_ip(each_standby_mgr))
         except:
             raise ExecutionFailureException(
-                "unable to convert a hostname to an IP address, monitoring host name: {}".format(monitoring_endpoint_ip))
+                "unable to convert a hostname to an IP address, for host: {}".format(failed_ip))
```

Comment:
"please enter the IP addresses of all the ceph-mgr with the '--monitoring-endpoint' flag"

Reply:
done...